test/data.c

```diff
@@ -414,6 +414,10 @@ char lan_logline[256];	    // defined in log_to_disk.c
 
 //////////////////
 
+/* resend call option, can be 0 (do not use), 1 (partial), 2 (full) */
+int resend_call = RESEND_NOT_SET;
```

Comment:
Let's move initialization to `init_variables()` in main.c. Background: it is needed to implement re-reading of configuration (see #233).

Reply:
Done. 971a01d fixes it.